nondex-gradle-plugin/src/main/groovy/edu/illinois/nondex/gradle/plugin/tasks/NonDexTest.groovy

```diff
@@ -11,7 +11,7 @@ class NonDexTest extends Test {
 
     void init(Configuration config) {
         project.extensions.create("nondexTest", NonDexTestExtension)
-        project.extensions.nondexTest.outPath = project.buildDir.absolutePath + "/out.jar"
+        project.extensions.nondexTest.outPath = project.buildDir.absolutePath + File.pathSeparator + "out.jar"
```

Comment:
I think you need File.separator; pathSeparator is what I used in the other PR. See the difference described at: http://stackoverflow.com/questions/5971964/file-separator-or-file-pathseparator